Error: Cannot find module 'node-fetch'

Recently, I started getting the `Error: Cannot find module 'node-fetch'` error when trying to use this module.  Turns out, we added a dependency on `node-fetch` at some point, but added it as a dev dependency 😐

--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "meow": "^5.0.0",
+    "node-fetch": "^2.2.0",
     "npm-package-arg": "^6.1.0",
     "tmp": "0.0.33",
     "update-notifier": "^2.5.0"
@@ -25,7 +26,6 @@
     "@types/update-notifier": "^2.2.0",
     "codecov": "^3.0.2",
     "gts": "^0.8.0",
-    "node-fetch": "^2.2.0",
     "nyc": "^12.0.2",
     "source-map-support": "^0.5.6",
     "tape": "^4.9.1",

